combineNodes.js

```diff
@@ -192,6 +192,40 @@ combineNodes = (...args) => {
 
 // ==========> END TESTS that calling a parent function will modify its child for nested objects <========== \\
 
+silo.prototype.subscribe = (component, name) => {
```

Comment:
Silo won't be the silo of the file that imports this because of the module that rescopes this file on it's own.